downloads/swan-lake-release-notes/swan-lake-preview4/RELEASE_NOTE.md

```diff
@@ -314,6 +315,18 @@ Executing a `fail` Statement will cause the control flow to transfer to the `on
     }
 ```
 
+##### Predeclared module prefixes
+
+The module prefix `t` (which is one of the `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml` types) has been predeclared as referring to the `lang.t` lang library module now.
```

Comment:
```suggestion
The module prefix `t` (which could be a `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml` types) has been predeclared as referring to the `lang.t` lang library module now.
```

Reply:
I think we should say 

```suggestion
The module prefix `t` (which could be one of `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml`) has been predeclared as referring to the `lang.t` lang library module now.
```

These are just prefixes/names here.

Reply:
"Where" should be used only to denote a location. Should be "which" in this case.